demo_notebook/visualisations demo notebook.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we need folium for the notebook\n",
```

Comment:
Overbodig --> is al geïnstalleerd als je de package installeert

Reply:
Weggehaald!